theseus_gui/src/pages/Browse.vue

```diff
@@ -259,13 +333,17 @@ async function refreshSearch() {
       limit: 1,
     }
   }
+
+  // Check for installed
   if (instanceContext.value) {
     for (val of rawResults.hits) {
       val.installed = await check_installed(instanceContext.value.path, val.project_id).then(
         (x) => (val.installed = x)
       )
     }
   }
+
+  // Set results
```

Comment:
useless comment

Reply:
In that logic all comments are useless

Reply:
Then remove it? I feel as though this is more personal preference than anything. It's a simple file change that can be done via the GitHub website. I don't need to pull out my code editor to make this change. I sectionize code and, to me, I feel as though this comment clarifies what the next section of this code is doing. If you feel it is unnecessary, you are more than welcome to remove it.